### DISCUSSION TITLE
[auth] Unauthorised domain error when calling sendEmailVerification

### DISCUSSION THREAD

**Discussion by xzilja** [invertase/react-native-firebase#6821] [Q&A]
I'm creating a user and sending verification email in the following way:

```
const { user } = await auth().createUserWithEmailAndPassword(email, password)
await user.sendEmailVerification({
  handleCodeInApp: true,
  url: `${ConstantsUtil.FIREBASE_DYNAMIC_LINK}/verify-email`
})
```

But this results in following error

> Error: [auth/unauthorized-domain] Domain not whitelisted by project
> NativeFirebaseError: [auth/unauthorized-domain] Domain not whitelisted by project

I assume it is due to me not adding any domains in firebase authentication dashboard, but I'm not sure what needs to be added there since we are running within native app?

**Comment by aka241193** [comment]
Was this solved?
